pkg/cli/flags/flags.go

```diff
@@ -15,10 +17,16 @@ type GlobalFlags struct {
 	Context   string
 	Namespace string
 	LogOutput string
+
+	readOnce  sync.Once
+	cliConfig *config.CLI
 }
 
 func (g *GlobalFlags) LoadedConfig(log log.Logger) *config.CLI {
```

Comment:
made it a once-only operation here

Reply:
I don't think this is necessary, as we already do this in `config.Read`: https://github.com/loft-sh/vcluster/blob/main/pkg/cli/config/config.go#L53
So, LoadedConfig is actually merely an "alias" for config.Read. I guess this the whole config handling is a bit inconsistent right now. WDYT of cleaning this in a follow-up after all the in-flight porting PRs are merged?

Reply:
Sure, can remove the once here.

> WDYT of cleaning this in a follow-up after all the in-flight porting PRs are merged? 🙂

Do you mean reducing the change in this PR to just ensuring the `platform add cluster` config is initialized?

Reply:
Sry, I expressed myself a bit misleading 😅. The change of this PR looks good. I would only drop the duplicate once handling

Reply:
Done :)